pkgs/servers/http/unit/default.nix

```diff
@@ -1,7 +1,6 @@
 { lib, stdenv, fetchFromGitHub, nixosTests, which
 , pcre2
 , withPython3 ? true, python3, ncurses
-, withPHP74 ? false, php74
```

Comment:
Maybe this should have been replaced by `php81`, don't you think?

Reply:
Well, possibly, however I leave that up to the `unit` maintainers to decide what they want to support or not. I was just in there to not break evaluation :wink: